tests/test_asbestos.py

```diff
@@ -288,3 +291,70 @@ def test_documentation_example() -> None:
         assert cursor.fetchmany() == [{"a": 1}, {"b": 2}]
         assert cursor.fetchmany() == [{"c": 3}, {"d": 4}]
         assert cursor.fetchmany() == [{"e": 5}]
+
+
+def test_fetchmany_with_bad_query() -> None:
+    with asbestos_cursor() as cursor:
+        cursor.execute(INVALID_QUERY)
+        assert cursor.fetchmany() == {}
```

Comment:
imho it should return a list, its fetchmany() not fetchone()

Reply:
Mmm. Good point.

Reply:
real snowflake cursor returns `TypeError: 'NoneType' object is not iterable` in that case. Empty list could be more useful in tests